roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -27,16 +27,31 @@ interface ComposePreviewTester<T : Any> {
   ) {
     interface TestLifecycleOptions
     data class JUnit4TestLifecycleOptions(
+      /**
+       * The TestRule factory to be used for the generated tests.
+       * You can use this to add custom behavior to the generated tests.
+       */
       // Used from generated tests
       @Suppress("unused") val testRuleFactory: () -> TestRule = { object : TestWatcher() {} },
```

Comment:
I do wonder whether there is some context that should be passed into the testRuleFactory? Or the methods on TestRule are enough?

Reply:
I thought the most likely scenario would be that we need to pass ComposeRule. I made adjustments to enable passing the ComposeRule.
https://github.com/takahirom/roborazzi/pull/445/commits/2f1255157b7b7c0180ab7116744db149f9acda06#diff-caa31d18b2a269ce09960b3943b693c5dcdd8c57ee34cd096b4a7ce27b6423adR13

Reply:
That looks like a great option. I've frequently needed to use the ComposeTestRule to interact with the UI.  nice.